tikv-configuration-file.md

```diff
@@ -1619,8 +1619,9 @@ Configuration items related to log backup.
 
 ### `file-size-limit` <span class="version-mark">New in v6.2.0</span>
 
-+ The size limit on log backup data. Once this limit is reached, the backup data is automatically flushed to external storage.
-+ Default value: 256MB
++ The size limit on log backup data to be stored.
++ Default value: 256MiB
++ Note: Generally, the value of `file-size-limit` is greater than the backup file size displayed in the storage system. This is because the backup files are compressed before being uploaded to the storage system.
```

Comment:
```suggestion
+ Note: Generally, the value of `file-size-limit` is greater than the backup file size displayed in external storage. This is because the backup files are compressed before being uploaded to the storage system.
```

Reply:
There is another `storage system`.
> ... uploaded to the `storage system`.